### PR TITLE
Added Descriptions for newly added chips

### PR DESCRIPTION
**Pull request by Alexa-RR**

### CODE REVIEW

File: misc/chip_data.json [hunk 1]
```diff
@@ -3044,7 +3044,7 @@
     },
     "2ee5a268-9d44-4326-8a34-ad126a3218f2": {
       "ReadonlyName": "Get Player By Account Name",
-      "Description": "",
+      "Description": "Gets the player vaule type based by their Account Name (e.g My Account name is Alexa and if I input my Account Name in the form of a string, The output would be my player value type.)",
```
[the remark this answers is not in the document]
Comment (Alexa-RR) [reply misc/chip_data.json]: Fixed :)

File: misc/chip_data.json [hunk 2]
```diff
@@ -1114,7 +1114,7 @@
       ]
     },
     "14ebde7a-76c0-47df-b0dd-01b1daa50821": {
-      "ReadonlyName": "No Name",
+      "ReadonlyName": "Circuit Board",
```
[the remark this answers is not in the document]
Comment (Alexa-RR) [reply misc/chip_data.json]: Changed it back to `No Name`

**Review by tyleo-rec — APPROVED** [review]

**Comment by Alexa-RR** [comment]
You forget to merge the pull request XD

**Comment by Alexa-RR** [comment]
:)

**Pull request merged**